source/app/service-providers/cli-provider.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @ignore
+ * BEGIN HEADER
+ *
+ * Contains:        cli-provider class
+ * CVM-Role:        Service Provider
+ * Authorr:         Felix Nüsse
+ * License:         GNU GPL v3
+ *
+ * Description:     This class handles the cli-arguments.
+ *                  It can be used to query arguments, as
+ *                  long as they are defined in the options object.
+ *
+ * END HEADER
+ */
+
+import { app } from 'electron'
+import path from 'path'
+
+export const DATA_DIR: string = 'data-dir'
+export const DISABLE_HARDWARE_ACCELERATION: string = 'disable-hardware-acceleration'
+export const CLEAR_CACHE: string = 'clear-cache'
+export const LAUNCH_MINIMIZED: string = 'launch-minimized'
+
+export function getCLIArgument (key: string): any {
```

Comment:
Should we make the return type more specific in that it will return either a `string` or a `boolean` (for existing CLI arguments) or `undefined` if it doesn't exist?